glean-core/ios/Glean/Config/Configuration.swift

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/// The Configuration struct describes how to configure Glean
+///
+/// - parameters:
+///   * serverEndpoint: A `String` representing the server the pings are sent to.  This should only be changed
+///     for tests
+///   * userAgent the user agent used when sending pings, only to be used internally.
```

Comment:
Above it has a colon, here it doesn't. One style here would be good, whatever the doc parser parsers right

Reply:
Ahh good catch!